samples/test/steps/when.ts

```diff
@@ -151,20 +153,30 @@ When(
   'she selects the {string} factor',
   async (authenticator: string) => {
     let authenticatorKey;
```

Comment:
Changed to switch/case to get rid of the following lint error - 
`error  Async arrow function has a complexity of 8. Maximum allowed is 7  complexity`